doc/rst/source/gmt.conf.rst

```diff
@@ -376,33 +375,33 @@ GMT Miscellaneous Parameters
         Transform in the West), **accelerate** (Use the Accelerate Framework
         under OS X; Note, that the number of samples to be processed must be
         a base 2 exponent), **kiss**, (Kiss FFT), **brenner** Brenner Legacy
-        FFT [auto].
+        FFT [default is **auto**].
         FFTW can "learn" how to optimally compute Fourier transforms on the
         current hardware and OS by computing several FFTs and measuring
         their execution time. This so gained "Wisdom" will be stored in and
         reloaded from the file fftw_wisdom_<hostname> in **$GMT_USERDIR** or, if
         **$GMT_USERDIR** is not writable, in the current directory. To use this
-        feature append *planner_flag*, which can be one of *measure*,
-        *patient*, and *exhaustive*; see FFTW reference for details. The
-        default FFTW planner flag is *estimate*, i.e., pick a (probably
-        sub-optimal) plan quickly. **Note**: If you need a single transform of a
+        feature append *planner_flag*, which can be one of **measure**,
+        **patient**, **exhaustive** and **estimate** which pick a (probably
+        sub-optimal) plan quickly [default is **estimate**]. See FFTW reference for details.
```

Comment:
```suggestion
        feature append *planner_flag*, which can be one of *measure*,
        *patient*, *exhaustive* and *estimate* which pick a (probably
        sub-optimal) plan quickly [default is *estimate*]. See FFTW reference for details.
```
Since these are secondary options, I think we should keep them as italics (similar to **extrapval**,*value*) in **GMT_EXPORT_TYPE**